db_scripts/mosip_prereg/ddl/user_details.sql

```
@@ -0,0 +1,31 @@
+    user_id UUID PRIMARY KEY DEFAULT gen_random_uuid(),
+    identifier_hash VARCHAR(128) NOT NULL UNIQUE,
+    identifier_encrypted TEXT NOT NULL,
+    cr_dtimes TIMESTAMP NOT NULL,
+    encrypted_dtimes TIMESTAMP NOT NULL
+);
+
+
+-- ========== CREATE INDEXES ==========
+
+CREATE INDEX IF NOT EXISTS idx_user_details_cr_dtimes
+  ON prereg.user_details (cr_dtimes);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)